docx4j-core/src/main/java/org/docx4j/openpackaging/io/Load.java

```diff
@@ -187,6 +188,7 @@ public static Part getRawPart(InputStream is, ContentTypeManager ctm, String res
 				        XMLInputFactory xif = XMLInputFactory.newInstance();
 				        xif.setProperty(XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, false);
 				        xif.setProperty(XMLInputFactory.SUPPORT_DTD, false); // a DTD is merely ignored, its presence doesn't cause an exception
+						xif.setProperty(XMLInputFactory.IS_REPLACING_ENTITY_REFERENCES, false);

```

Comment:
spacing not lining up

Reply:
![image](https://github.com/LegalSifter/docx4j/assets/47982752/b36875b4-3057-4b35-9443-c9f212ab23b8)

github might be having a moment because it lines up in my IDE

Reply:
ah nope, spaces, not tabs

Reply:
wait it's even more psychotic than that, it's both